oat-sa/tao-core
# Fix/tca 620 button link announcement

Related to : https://oat-sa.atlassian.net/browse/TCA-620
 
test cards announced twice: as link and as button
 
#### How to test
 
- prepare an instance of TAO
- prepare an any delivery 
- open deliveries list and navigate via keyboard to listen SR announces
- ensure test cards not announced as links
   
Companion PR :
 - [ ] https://github.com/oat-sa/extension-tao-delivery/pull/483
